docs/end-user-topics/language-support.rst

```diff
@@ -13,25 +13,29 @@ on the Dockstore site and in the Dockstore command-line utility.
 .. raw:: html
     :file: ../_static/language-support.html
 
-[0] Available in both classic and CWL Viewer modes
-
-[1] Available in both classic and WDL EPAM Pipeline Builder
+[1] Available in both classic and CWL Viewer modes
 
 [2] Does not support http(s) based imports. See :ref:`CGC
 Limitations <cgc-limitations>` for limitations.
 
-[3] Does not support file-path based imports. See :ref:`Terra
-Limitations <terra-limitations>` for limitations.
+[3] Does not support file-path based imports. See :ref:`BioData Catalyst
+Limitations <anvil-limitations>` for limitations.
+
+[4] Does not support file-path based imports. See :ref:`AnVIL
+Limitations <anvil-limitations>` for limitations.
 
-[4] Does not support file-path or http(s) based imports. See :ref:`DNAstack
+[5] Does not support file-path or http(s) based imports. See :ref:`DNAstack
 Limitations <dnastack-limitations>` for limitations.
 
-[5] All verified Dockstore WDL tools/workflows were tested successfully. However, we anticipate that more testing is needed for WDL workflows that use language features not contained within that dataset.
+[6] Does not support file-path based imports. See :ref:`Terra
+Limitations <terra-limitations>` for limitations.
+
+[7] There is no launch with Galaxy button in Dockstore, but Dockstore workflows can be searched and imported from within Galaxy.
```

Comment:
Nice addition

Sorry to add new work, but that this cell is a conditional yes but is yellow while all the other "nos" are yellow too makes me think either
1) this cell should be a new colour
2) "no" should be grey and this should be yellow

Reply:
Pushed changes with colors, please take a look.